src/utils/redirect.ts

```diff
@@ -5,7 +5,7 @@ import { db } from "~/server/db";
 
 export const authRedirect = async (
```

Comment:
We need to be careful with defaulting to /internal/login here because that's only for organizer portal redirects.

Reply:
yeah maybe we should have 2 functions for both functionalities and name them properly

Reply:
wait, were using a NotAuthroizedCard if the user is not permitted to acess organizer pages, so ill make this just a regular auth redirect if theyre not logged in and get rid of the user type check

Reply:
nothing important is using this function as of right now, so it should be fine